api/cases/case.spec.js

```diff
@@ -90,6 +106,9 @@ describe('case spec', () => {
             httpResponse = (resolve, reject) => {
                 resolve(caseData);
             };
+            eventsHttpResponse = (resolve, reject) => {
+                resolve([]);
```

Comment:
this test is not complete. Please add data to this resolve() -> account for two scenarios at least 1) includes casedata and case event data 2) includes only case data and no event data